app/Main.js

```diff
@@ -25,7 +25,10 @@ try {
 function runApp(initial_state) {
     console.log('Initial state', initial_state);
     const config = initial_state.offchain.config
-    steem.config.set('websocket', config.ws_connection_client);
+    if(process.env.USE_JSONRPC)
```

Comment:
this is using server env vars but this is the http_connection_client - the URI for the SPA to talk to, no?  Am I confused?